### PR TITLE
apps/http: add http download flashing stress test

### PR DESCRIPTION
**Pull request by pillip8282**
**Caution**: To save your time please do not review this PR. it's test code so it'll be closed later.

1) write a server address to hello_main.c REQ_URL
2) build
3) run and connect to wi-fi by wm_test
4) test program name hello
  - usage: hello count
   ex. TASH> hello 2

### CODE REVIEW

File: apps/examples/hello/hello_main.c [hunk 1]
```diff
@@ -55,18 +55,138 @@
...
+		//printf("written (%d/%d)\n", len, buf_size);
+	}
+	return 0;
+}
+
+static size_t curl_callback(void *contents, size_t size, size_t nmemb,
+							void *userp)
+{
+	printf("size %lu  nmemb %lu\n", size, nmemb);
+	struct message *msg = (struct message *)userp;
```
Comment (sangam-swami) [inline apps/examples/hello/hello_main.c:103]: can we validate the pointer userp

File: apps/examples/hello/hello_main.c [hunk 2]
```diff
@@ -55,18 +55,138 @@
...
+
+struct message {
+	int fd;
+	int received;
+	int check;
+};
+
+static int g_num = 0;
+
+static int _write_flash(int fd, char *buf, int buf_size)
```
Comment (sangam-swami) [inline apps/examples/hello/hello_main.c:78]: if return if not used, we can declare void

File: apps/examples/hello/hello_main.c [hunk 3]
```diff
@@ -55,18 +55,138 @@
...
+}
+
+static size_t curl_callback(void *contents, size_t size, size_t nmemb,
+							void *userp)
+{
+	printf("size %lu  nmemb %lu\n", size, nmemb);
+	struct message *msg = (struct message *)userp;
+	size_t left = size * nmemb;
+	int offset = 0;
+	msg->received += (size * nmemb);
```
Comment (sangam-swami) [inline apps/examples/hello/hello_main.c:106]: can we update msg at the end of write completion

File: apps/examples/hello/hello_main.c [hunk 4]
```diff
@@ -55,18 +55,138 @@
...
+	}
+	curl_global_cleanup();
+	close(fd);
+	return 0;
+}
+
+void _wt_process(int argc, char *argv[])
+{
+	for (int i = 0; i < g_num; i++) {
+		int res = _download();
```
Comment (sangam-swami) [inline apps/examples/hello/hello_main.c:170]: res is not used we can remove? and change _download to void

**Comment by pillip8282** [comment]
Please do not review this PR. it's test code so it'll be closed later.

**Pull request closed without merging**